pkg/scheduler/framework/plugins/noderesources/fit_test.go

```diff
@@ -693,9 +688,15 @@ func TestEnoughRequests(t *testing.T) {
 			ctx, cancel := context.WithCancel(ctx)
 			defer cancel()
 
+			if test.draExtendedResourceEnabled {
+				featuregatetesting.SetFeatureGateDuringTest(t, utilfeature.DefaultFeatureGate, features.DRAExtendedResource, true)
```

Comment:
Move it just below `t.Run` and `if` can be omitted:
```suggestion
				featuregatetesting.SetFeatureGateDuringTest(t, utilfeature.DefaultFeatureGate, features.DRAExtendedResource, test.draExtendedResourceEnabled)
```